### PR TITLE
Feature/databricks support

### PR DESCRIPTION
**Pull request by agnessnowplow**
Adding the Databricks support as far as I've got to with it before my holidays. 

It needs revision on wording and perhaps some more help for users when it comes to the UI to navigate having different warehouses. (perhaps a decision matrix of some sort with all the options for each chapter?)

I also left some currently superfluous filters within the notebook as it does not seem to work on dates but perhaps there is a way to solve this.

### CODE REVIEW

File: content/next_steps/next_steps_2.md [hunk 1]
```diff
@@ -14,3 +18,14 @@ FROM NEW_SCHEMA_NAME.snowplow_web_sessions
 WHERE START_TSTAMP BETWEEN DATEADD(day, -7, GETDATE()) AND  DATEADD(day, -1, GETDATE())
 ```
 
+{{% /tab %}}
+
+{{% tab name="Databricks Notebook" %}}
+
+Assuming you have already imported the IPython notebook provided in the chapter about visualisations, all you have to do is change the schema within the queries used to generate the graphs.
+
+You can do this quickly by using the `Edit/ Find and replace` functionality.
```
Comment (Craig-Mace) [inline content/next_steps/next_steps_2.md:27]: Change 'Edit/ Find and replace' to 

'Edit / Find and replace'

File: content/upload/upload_2.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+***
+
+#### **Step 2:**  Upload the SAMPLE_DATA.CSV file
+2.1 Select `+Create` -> `table` to get started.
+
+2.2 Locate the sample_events.csv file and click `Upload`. It should take a minute or two.
+
+2.3 Once you see the preview, choose the catalog, the schema (atomic) and change the name to sample_events**_base**.
+
+2.4 Go to `advanced attributes` and change the column delimiter to tab then X out of the window. After this the data reloads, which should now be separated correctly column by column.
```
Comment (Craig-Mace) [inline content/upload/upload_2.md:31]: Change "which should now"
to
"which will now"

**Review by Craig-Mace — COMMENTED** [review]
Small minor changes and suggestions

**Comment by agnessnowplow** [comment]
closing this as it became obsolete

**Pull request closed without merging**